### PR TITLE
Identity Section Improvements

### PR DESCRIPTION
**Pull request by Identitywoman**
I at Glen's request wrote about identity systems/documents in a WEIRD historical context.

### CODE REVIEW

**Comment by pluralitybook** [comment]
Yup!  Reviewing now.

 

From: Kaliya - Identity Woman ***@***.***> 
Sent: Tuesday, July 11, 2023 2:03 PM
To: pluralitybook/plurality ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [pluralitybook/plurality] Identity Section Improvements (PR #49)

 

I at Glen's request wrote about identity systems/documents in a WEIRD historical context.

  _____  


You can view, comment on, or merge this pull request online at:


  https://github.com/pluralitybook/plurality/pull/49


Commit Summary


*	34f6872 <https://github.com/pluralitybook/plurality/pull/49/commits/34f6872d489d4b6b5e9e7d41b87c1d5449c4aa11>  Update LostDao.md
*	547d329 <https://github.com/pluralitybook/plurality/pull/49/commits/547d32939f5392fba99a99dc0ccec63d5f6847d4>  Update LostDao.md
*	edd4d16 <https://github.com/pluralitybook/plurality/pull/49/commits/edd4d1671c6897c6310a0f606a0d6bb4d914ab7b>  Update LostDao.md
*	1ce709d <https://github.com/pluralitybook/plurality/pull/49/commits/1ce709dbaa7dcf605f1b8bdb904d064580e3cac6>  Merge pull request #1 from Identitywoman/Identitywoman-patch-1
*	162c486 <https://github.com/pluralitybook/plurality/pull/49/commits/162c48619154c395be02e7e47c7026f1cbdc1c61>  Update LostDao.md
*	fb9f4d1 <https://github.com/pluralitybook/plurality/pull/49/commits/fb9f4d11d98de4243d0310fb08d9022ed4fb57ad>  Update LostDao.md


File Changes 


(1 <https://github.com/pluralitybook/plurality/pull/49/files>  file)

*	M LostDao.md <https://github.com/pluralitybook/plurality/pull/49/files#diff-8b176bcbc6192df561bbe29cd2fe7da484fec574651280642c90923b86a11dc6>  (194) 


Patch Links:


*	https://github.com/pluralitybook/plurality/pull/49.patch
*	https://github.com/pluralitybook/plurality/pull/49.diff

**Review by pluralitybook — CHANGES_REQUESTED** [review]
In general I really like what you did here.  I'd like to have more references added, but someone else can do this later, and there are parts that don't quite belong here that I will move to a different chapter.  But the biggest problem is that I think something in your editor auto-removed the indent of every paragraph in the document and I don't want to merge your PR as it will force me to redo this globally.  Could you fix this before I merge the PR?

**Comment by Identitywoman** [comment]
I did that removal of the indents.
Best practice for mark down is not to have indents.

the weird characters were driving me nuts.
If you really want me to add in spaces I will.
- Kaliya

**Comment by pluralitybook** [comment]
If you want to submit a global formatting change to all chapters, I will consider this, but it doesn’t belong in this local PR, or so bad hygiene.Infinite diversity in infinite combinationshttps://www.RadicalxChange.org.On Jul 11, 2023, at 5:10 PM, Kaliya - Identity Woman ***@***.***> wrote:﻿
I did that removal of the indents.

Best practice for mark down is not to have indents.



the weird characters were driving me nuts.

If you really want me to add in spaces I will.

- Kaliya





On Tue, Jul 11, 2023 at 11:26 AM pluralitybook ***@***.***>

wrote:

File: LostDao.md [hunk 1]
```diff
@@ -41,204 +41,215 @@ Industry and inventions in technology, for example, create means which alter the
...
 
+Who you where and where you fit was not "obious" based on your kin relations anymore. So, as people began to move around more and new institutions formed paper based systems to document who belonged to them emerged, who was baptized by the church, who was a resident of a town, who was a member in a guild, who was a soldier in the army, who was a patient at the hospital etc.  Identity systems in liberal democratic countries are rooted in historic practices originating from the church practices of keeping baptismal records in a log book. Beginning in the 1500s there was a shift over several centuries towards becoming state run systems where births were registered and birth certificates were issued to parents. This document, the birth certificate, is still the root breeder document that all other state issued identity documents (drivers license, national ID, tax/pension number, passports) are derived.
```
Comment (audreyt) [inline LostDao.md:48]: `obious` => `obvious`

**Pull request closed without merging**